ats/test/HelloATS/create_test_ats.py

```diff
@@ -1,21 +1,22 @@
-#!/usr/apps/ats/7.0.1/bin/python
+#!/usr/bin/env python3
 """Create a test.ats file."""
 import itertools
 import os
+import sys
 
 
 def get_file_header():
     """Returns ATS test file header. Contains imports, glue, and checker."""
     ats_log_checker_path = os.path.join(
         os.path.dirname(os.path.abspath(__file__)), 'ats_check_log.py')
     if not os.path.isfile(ats_log_checker_path):
-        exit(1)
+        sys.exit(1)
```

Comment:
(Also out of scope but) an error message could be helpful

Reply:
Agreed. Mostly updated this a part of what pylint was complaining about.